### ISSUE TITLE
Issue with output to an external hard drive

### ISSUE THREAD

**Issue by saka83838383** [AlphaSlayer1964/kemono-dl#112]
As in the lastest version, there is an output template to specify the personalize the output link, which is awsome. To me, I always use -o "/Volumes/My Drive Name" to select my external hard drive. Now when I use the following input, it created a folder named 'Volumes' and a sub folder 'My Drive Name' under the kemono-dl instead of output to my hard drive.

My current input:
`python3 kemono-dl.py --cookies "cookie.txt" --links https://coomer.party/onlyfans/user/xxx --dirname-pattern "/Volumes/My Drive Name/{service}/{username}" --filename-pattern "[{published}] {title}\{index}_{filename}.{ext}"  `

Thanks!

**Comment by AlphaSlayer1964** [comment]
I might have to send you a modified main.py for you to test because I don't use Linux.

**Comment by afterdelight** [comment]
try to put your real external drive folder path instead of /Volumes/My Drive Name

**Comment by saka83838383** [comment]
> 

yep, sorry to not mentioning that I'm using mac, which might be the reason.

**Comment by ProblemCauser** [comment]
I have the same problem as OP. Cannot write data to my mounted NAS on MacOS. Very unfortunate seeing as the rest of the software works perfectly well for me.
